sdk/voiceservices/Azure.ResourceManager.VoiceServices/src/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,9 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Runtime.CompilerServices;
+
+[assembly: Azure.Core.AzureResourceProviderNamespace("VoiceServices")]
```

Comment:
The value should be `Microsoft.VoiceServices`

Reply:
Thanks, I've fixed it